src/bin/solang.rs

```diff
@@ -177,6 +177,12 @@ fn main() {
                             .short('g')
                             .long("generate-debug-info")
                             .hide(true),
+                    )
+                    .arg(
+                        Arg::new("LOGAPICALLS")
+                            .help("Log the return codes of runtime API calls in the environment")
```

Comment:
How do you feel about `syscalls` rather than `runtime API calls`? I am wondering if that is an solana-ism though, not sure if the `seal_*` calls are called syscalls in the Substrate world.

Reply:
Hmm, I don't really like `syscalls`. I see those more as API calls into the on, off or whatever chain environment of the target?